README.md

```diff
@@ -186,4 +184,4 @@ If everything works, commit the updated version, then tag a new repository relea
 
 ## Author
 
-This operator was built in 2019 by [Jeff Geerling](https://www.jeffgeerling.com), author of [Ansible for DevOps](https://www.ansiblefordevops.com) and [Ansible for Kubernetes](https://www.ansibleforkubernetes.com).
+This operator was originally built in 2019 by [Jeff Geerling](https://www.jeffgeerling.com), author of [Ansible for DevOps](https://www.ansiblefordevops.com) and [Ansible for Kubernetes](https://www.ansibleforkubernetes.com) and is now maintained by the Ansible Team
```

Comment:
You can drop the book links here. For now, at least, I'm happy with just a link / recognition, and this can fall off at some point.